.vscode/launch.json

```diff
@@ -1,6 +1,15 @@
 {
     "version": "0.2.0",
     "configurations": [
+         {
+            "type": "julia",
+            "request": "launch",
+            "name": "Julia: current file",
+            "program": "${file}",
+            "stopOnEntry": false,
+            "cwd": "${workspaceFolder}",
```

Comment:
Will this automatically activate `--project=core`? Do we want to add that, so that all the packages are correct?

Reply:
Yeah I think the current `"juliaEnv": "${command:activeJuliaEnvironment}"` is fine since it allows users to control it, and still has a good default setting of the `root` environment.